umh-core/integration/docker_test_helpers.go

```
@@ -0,0 +1,363 @@
+}
+
+// cleanupConfigFile removes the test-specific config file
+func cleanupConfigFile() {
+	if configFilePath != "" {
+		// Just attempt to remove it, ignoring errors
+		os.Remove(configFilePath)
+	}
+}
+
+// getContainerIP returns the IP address of a running container
+func getContainerIP(container string) (string, error) {
```

golangci-lint (1.64.8)

func `getContainerIP` is unused

(unused)